assets/app/views/browse/build.html

```diff
@@ -105,10 +105,20 @@
                         name="build.metadata.name"
                         context="projectContext"
                         options="logOptions"
-                        status="build.status.phase"
-                        time-start="build.status.startTimestamp | date : 'short'"
-                        time-end="build.status.completionTimestamp | date : 'short'"
                         run="logCanRun">
+                        <div row mobile="column" tablet="column">
+                          <div>Build status:&nbsp;</div>
```

Comment:
I'd rather not have them if we can avoid it. They seem to add weird margins around the icons.